### ISSUE TITLE
Doesn't work in iOS

### ISSUE THREAD

**Issue by sumi-svmx** [aarongrider/vision-camera-ocr#6]
I tried the project in example folder, while it works in android, it always returns empty data in iOS, and shows this error in logs.

`VisionCamera.captureOutput(_:didOutput:from:): The Frame Processor took so long to execute that a frame was dropped.`

**Comment by luismiguelmena** [comment]
I have the same problem, works good on Android but not on iOS.
Any solution?

**Comment by smisaacs** [comment]
runs perfectly for me on iOS. iPhone XS, using Expo SDK45.

**Comment by sumi-svmx** [comment]
@smisaacs Which version of iOS did you run it on? For me the text always return empty with iOS v14.1. I have heard it works fine in v15.4.1 Have you set any format/ preset different from the one given in /example?

**Comment by smisaacs** [comment]
yes. 15.4.1.  Only played with framerates and targeting lines vs blocks.  Works in all configurations though.

**Comment by aarongrider** [comment]
Interesting, this may be a limitation of your hardware and / or Vision Camera. Can you file a bug with vision camera?

**Comment by dinhthi1610** [comment]
hello @aarongrider please help to check on ios 14, i tried but it not work
Thanks

**Comment by MSchmidt** [comment]
Seeing the same. There is an issue with iOS 14 only. Maybe 13 and older as well? There are still people with iOS 14 out there. I played with the orientation but doesn't change anything. Result still empty.

**Comment by CodingWithNobody** [comment]
Check this https://www.youtube.com/watch?v=kOYP2Xun_6o&list=PLQhQEGkwKZUrempLnmxjt7ZCZJu1W3p2i&index=77
